src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -56,6 +56,7 @@
 #include "JsonUtils.h"
 #include "HashUtils.h"
 #include "TerminalWarnings.h"
+#include "../WinRTUtils/inc/LibraryResources.h"

```

Comment:
Huh, whatcha need this for? I don't see any `RS_` in this file.

Reply:
It's in the macros! See `ActionArgsMagic.h`